eval/src/vespa/eval/eval/value_codec.cpp

```diff
@@ -154,6 +153,9 @@ struct ContentDecoder {
     template<typename T>
     static std::unique_ptr<Value> invoke(nbostream &input, const DecodeState &state, const ValueBuilderFactory &factory) {
         std::vector<vespalib::stringref> address(state.num_mapped_dims);
+        if (state.num_blocks * state.subspace_size * sizeof(T) > input.size()) {
+            throw IllegalStateException("serialized input too big");
```

Comment:
or is it too small...?

Reply:
Also some information about the sizes in the exception message would be nice.